formatter_filters/cloudevents/formatter_filter.go

```diff
@@ -211,3 +229,43 @@ func (f *FormatterFilter) Type() eventlogger.NodeType {
 func (f *FormatterFilter) Name() string {
 	return NodeName
 }
+
+func (f *FormatterFilter) sign(ctx context.Context, e *Event, enc *json.Encoder, buf *bytes.Buffer) error {
+	const op = "cloudevents.(FormatterFilter).sign"
+	if e == nil {
+		return fmt.Errorf("%s: missing event: %w", op, eventlogger.ErrInvalidParameter)
+	}
+	if enc == nil {
+		return fmt.Errorf("%s: missing encoder: %w", op, eventlogger.ErrInvalidParameter)
+	}
+	if buf == nil {
+		return fmt.Errorf("%s: missing buffer: %w", op, eventlogger.ErrInvalidParameter)
+	}
+	if f.Signer != nil {
+		bufHmac, err := f.Signer(ctx, buf.Bytes())
+		if err != nil {
+			return fmt.Errorf("%s: unable to hmac-sha256: %w", op, err)
```

Comment:
Since the `Signer` is configurable shouldn't the error message be `unable to sign` rather than calling out `hmac-sha256`

Reply:
ty!